resilience4j-spring-boot/src/main/java/io/github/resilience4j/circuitbreaker/autoconfigure/CircuitBreakerMetricsAutoConfiguration.java

```diff
@@ -39,9 +36,7 @@
 @AutoConfigureBefore(MetricRepositoryAutoConfiguration.class)
 public class CircuitBreakerMetricsAutoConfiguration {
     @Bean
-    public CircuitBreakerMetrics registerCircuitBreakerMetrics(CircuitBreakerRegistry circuitBreakerRegistry, MetricRegistry metricRegistry){
-        CircuitBreakerMetrics circuitBreakerMetrics = CircuitBreakerMetrics.ofCircuitBreakerRegistry(circuitBreakerRegistry);
-        metricRegistry.registerAll(circuitBreakerMetrics);
```

Comment:
I think your described solution only works with Spring Boot 2 and Micrometer, but not with Spring Boot 1 and Dropwizard Metrics, or?

Reply:
I think in Spring Boot 1 it's still necessary to register all Metrics from a `MetricSet`

Reply:
Thanks, I'll take a look at it.

Reply:
Thanks for catching that breaking change!  I've been using Micrometer's support for Spring Boot 1.5 and completely missed that this was registering it with a `MetricRegistry` and not a `MeterRegistry`.

I reverted the Spring Boot 1.x change as there should be no issue when using Dropwizard Metrics since there are no tags :)